mobile/views/LocationHistory/LocationHistory.tsx

```diff
@@ -60,7 +52,7 @@ const LocationHistory = () => {
   useEffect(() => {
     const fetchLocations = async () => {
       const locations = await getBikeLocationHistory();
-      setLocationArray(locations);
+      setLocationArray(locations.reverse());
```

Comment:
Should check location for undefined, since calling reverse will cause app to crash

Reply:
u right i forgot about that. in theory any new device should be instantiated with an empty array rather than undefined location array but better safe than sorry